pytest.ini

```diff
@@ -19,5 +19,8 @@ filterwarnings =
     # numpy uses distutils which is deprecated
     ignore:The distutils.* is deprecated.*:DeprecationWarning
     ignore:`sharded_jit` is deprecated. Please use `pjit` instead.*:DeprecationWarning
+    # print message for compilation_cache_test.py::CompilationCacheTest::test_cache_read/write_warning 
```

Comment:
nit: capitalize first letter of comment (unless it's a variable name or something else that's always lowercase) (`Print`)

Reply:
done